src/winstanley/wdl.bnf

```diff
@@ -65,7 +65,7 @@ command_var_option ::= command_var_option_sep|command_var_option_default|command
 
 task_outputs ::= OUTPUT LBRACE output_kv* RBRACE
 output_kv ::= type_e IDENTIFIER EQUAL expression {mixin="winstanley.psi.impl.WdlNamedElementImpl" implements="winstanley.psi.WdlNamedElement" methods=[getName getNameIdentifier setName]}
-runtime_block ::= RUNTIME map
+runtime_block ::= RUNTIME LBRACE kv* RBRACE
```

Comment:
Is this right? It doesn't seem necessary since `map ::= LBRACE kv* RBRACE`

Reply:
As far as I remember It had strange implications on formatting of the runtime block (like indent inside the runtime block was relative to LBRACE and not the `runtime` word itself for some reason). This change fixed that problem.